src/lava/magma/core/model/py/ports.py

```diff
@@ -516,9 +516,19 @@ def write(
     # TODO: (PP) This should be optimized by a proper CSPSendPort wait
     def wait(self):
         """Method which block execution until (all) previous write requests have
```

Comment:
No need to say that this is a method. 

"Blocks execution until receipt of prior 'write' commands (sent from RefPort to VarPort) have been acknowledged..."

Reply:
Done